logprep/processor/requester/rule.py

```diff
@@ -0,0 +1,186 @@
+"""
+Requester
+=========
+
+The Requester is configured by the keyword :code:`requester`.
+It can be used to trigger external systems via web request or enrich eventdata by external
+apis.
+
+A speaking example for event enrichment via external api:
+
+..  code-block:: yaml
```

Comment:
Can you add to this example how the raw response from the api call would look like? I think that could enrich the understanding of this example.

Reply:
done